Makefile

```diff
@@ -173,6 +173,26 @@ swagger: config/swagger.yaml $(SWAGGER_CODEGEN_CLI)
 swagger-docs:
 	(cd ./apps/aehttp && $(MAKE) swagger-docs);
 
+SWAGGER_STATUS = $(shell cd /tmp/epoch-api-docs/ && git status --porcelain)
```

Comment:
This process is CI only, let's not pollute the main Makefile with it. `deployment/deploy_api_docs.sh` sounds much better place to me.

Reply:
duty noted